doc/RecordingControl.xml

```diff
@@ -2221,9 +2221,58 @@ secfrac = "." 1*6DIGIT</programlisting>
                 <para>Integer</para>
               </entry>
             </row>
+            <row>
+              <entry>
+                <para>
+                  <literal>Timescale</literal>
+                </para>
+              </entry>
+              <entry>
+                <para>The <literal>timescale</literal> (or clock) from the segment in hertz</para>
+              </entry>
+              <entry>
+                <para>Integer (See <xref linkend="_refTimescaleAttribute"/>)</para>
+              </entry>
+            </row>
+            <row>
+              <entry>
+                <para>
+                  <literal>SeekTable</literal>
+                </para>
+              </entry>
+              <entry>
+                <para>Seek table to efficiently locate file fragments and seek in the file</para>
+              </entry>
+              <entry>
+                <para>String (See <xref linkend="_refSeekTableAttribute"/>)</para>
+              </entry>
+            </row>
           </tbody>
         </tgroup>
       </table>
+      <section xml:id="_refTimescaleAttribute">
+        <title>Timescale Attribute</title>
+        <para>
+          The value of the <literal>timescale</literal> property of the <literal>MediaHeaderBox</literal> in the segment as defined in ISO/IEC 14496-12:2012 section 8.4.2. If there are multiple tracks in the segment each of these boxes shall all have the same <literal>timescale</literal> value. The <literal>MovieHeaderBox</literal> shall also have the same <literal>timescale</literal> value.
+        </para>
+      </section>
+      <section xml:id="_refSeekTableAttribute">
+        <title>SeekTable Attribute</title>
+        <para>
+          This seek table attribute is a string that contains the 'mfra' (Movie Fragment Random Access) box corresponding to the associated ISO file. It is encoded in base64 with padding according to RFC 4658. The binary format of the box is defined in ISO/IEC 14496-12:2012 section 8.8.9. This optional box has to be the last one at the end of the file. It is copied in the object metadata to allow the receiver to efficiently locate file fragments and seek in the file without the need to access the file itself.
```

Comment:
Propose to replace the last sentence `It is copied in the object metadata to allow the receiver to efficiently locate file fragments and seek in the file without the need to access the file itself.` by

`The optionally condensed copy allows the receiver to efficiently locate file fragments and seek in the file without the need to access the file itself.`